### DISCUSSION TITLE
energy deviation in LAMMPS simulation

### DISCUSSION THREAD

**Discussion by marian-code** [deepmodeling/deepmd-kit#1190] [Q&A] [answered]
I have noticed that in the new v2 version the energy deviation from lammps output file is gone. It was replaced by virial deviation which is undoubtedly useful, but so is the energy deviation. Is there some specific reason for this? I was not able to find any information on this in the respective pull request #594.

**Comment by njzjz** [comment]
The energy deviation is not as accurate as force deviation or viral deviation -- that's the reason why it's deprecated.

**Comment by amcadmus** [comment, marked as answer]
There is an arbitrariness in the energy decomposition. For example, we have molecule AB in the system, adding an arbitrary constant to E_A and subtract the same constant from E_B does not change the total energy of the system. Therefore, the model deviation of atomic energy is not a good indication for the quality of the model. The atomic energy deviation is printed in deepmd-kit v1.x, but it is not intended to be used in any case. That's why it is removed in v2.

**Reply by marian-code** [reply]
Ok, the reasoning is clear. That is what I have suspected. But I could have been mentioned in the docs to avoid confusion.
